test/python/gemm/frost/test_block_scale_matmul.py

```
@@ -1619,62 +1673,85 @@ def _pretend_sm107(monkeypatch):
+
+
+def test_catalog_carries_both_mma_k_widths():
+    """The 64-byte MMA K is a geometry AXIS of the sm100 family, crossed over the
+    same M / N / cluster enumeration as the 32-byte one — not a family of its own."""
+    sm100 = [c for c in CATALOG if c.pipeline == "sm100"]
+    by_width = {32: [], 64: []}
+    for c in sm100:
+        by_width[c.mma_tile_k_bytes].append(c)
+    assert len(by_width[32]) == len(by_width[64]) > 0
+    # Same geometry, different width -> same name modulo the MMA-inst K token.
+    strip = lambda c: (c.cta_tile_mn, c.cga_size_mn, c.cta_tile_k_bytes, c.mma_tile_m, c.mma_size_m)
```

Ruff (0.16.2)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `strip` as a `def`

(E731)